Description: Fixes opacity not set on tab creation.
Origin: http://bazaar.launchpad.net/~dabisu/sakura/sakura/revision/417
Forwarded: not needed
Bug: https://bugs.launchpad.net/sakura/+bug/969294

=== modified file 'src/sakura.c'
Index: sakura/src/sakura.c
===================================================================
--- sakura.orig/src/sakura.c	2012-04-17 21:22:00.608761528 -0400
+++ sakura/src/sakura.c	2012-04-17 21:22:00.672761829 -0400
@@ -2598,7 +2598,8 @@
 	free(cwd);
 
 	/* Configuration for the newly created terminal */
-
+	GdkColor white={0, 255, 255, 255};
+	vte_terminal_set_color_background(VTE_TERMINAL (term->vte), &white);
 	vte_terminal_set_backspace_binding(VTE_TERMINAL(term->vte), VTE_ERASE_ASCII_DELETE);
 	vte_terminal_set_colors(VTE_TERMINAL(term->vte), &sakura.forecolor, &sakura.backcolor,
 	                        sakura.palette, PALETTE_SIZE);
